lizmap/www/js/map.js

```diff
@@ -3754,9 +3754,9 @@ var lizMap = function() {
         }
       });
 
-      printLayers.reverse();
-      styleLayers.reverse();
-      opacityLayers.reverse();
+      //printLayers.reverse();
+      //styleLayers.reverse();
+      //opacityLayers.reverse();
```

Comment:
Hi @xcaeag you can't comment this lines, because this correction is available for QGIS 2.18 and not for 2.14.
The fix has to be more complex.